Added Vineeth's picture to the card

Aniket-Mishra opened the pull request:
Added Vineeth's picture to the card
Mobile View looks file here, So does the large screen view
![added-vineeth](https://user-images.githubusercontent.com/38854722/88939791-94bb6500-d2a4-11ea-8296-f3e83620ecb3.png)
![added-vineeth-phone-1](https://user-images.githubusercontent.com/38854722/88939799-97b65580-d2a4-11ea-8c87-930e7235f22f.png)
![added-vineeth-phone-2](https://user-images.githubusercontent.com/38854722/88939800-984eec00-d2a4-11ea-8547-d4583ffd5ebe.png)




Changed designation of Rafael from:
Coach - Platzi, CEO - DevHood
to
CEO - DevHood, Coach Platzi
![rafael-designation](https://user-images.githubusercontent.com/38854722/88939907-b61c5100-d2a4-11ea-92c6-1ad15916c1fa.png)

swaaz committed:
fixed everything

swaaz committed:
added prizes

swaaz commented:
![image](https://user-images.githubusercontent.com/42874695/88960870-c7278b00-d2c1-11ea-8bc7-1882955af994.png)
![image](https://user-images.githubusercontent.com/42874695/88960956-eb836780-d2c1-11ea-8946-89924960ec5a.png)
![image](https://user-images.githubusercontent.com/42874695/88960965-f0481b80-d2c1-11ea-96b4-5d7f4b0fe3e4.png)

Aniket-Mishra committed:
Added Vineeth's picture to the card

Changed designation of Rafael from:
Coach - Platzi, CEO - DevHood
to
CEO - DevHood, Coach Platzi
Mobile View looks file here, So does the large screen view

Aniket-Mishra committed:
Name shift - aman sanduja -> Shivay Lamba
